build-run.ps1

```diff
@@ -33,18 +33,13 @@ cd $PSScriptRoot
 $repoFolder = $PSScriptRoot
 $env:REPO_FOLDER = $repoFolder
 
-if ($env:BuildRunner -eq "MyGet"){
-    Remove-Item -Force -Recurse (Join-Path $PSScriptRoot (Join-Path "test" "Pomelo.EntityFrameworkCore.MySql.PerfTests"))
-}
-
-$koreBuildZip="https://github.com/PomeloFoundation/KoreBuild/releases/download/1.0.0/KoreBuild-1.0.0.zip"
+$koreBuildZip="https://github.com/aspnet/KoreBuild/archive/dev.zip"
```

Comment:
Right now we are just using the install part of KoreBuild, which installs the latest preview environment, `dotnet-dev-win-x64.1.0.0-preview2.1-003155.zip`.  This is the same version as listed on the [Announcing .NET Core 1.1 Preview 1](https://blogs.msdn.microsoft.com/dotnet/2016/10/25/announcing-net-core-1-1-preview-1/) page.